api/nakadi-event-bus-api.yaml

```diff
@@ -479,7 +475,34 @@ paths:
                 Number of seconds the client ought to wait before making a follow-up request.
               type: number
 
-  /event-types/{name}/partitions:
+  '/event-types/{name}/schemas':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        List of schemas ordered from most recent to oldest.
+    responses:
+        '200':
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/EventTypeSchema'
+
+  '/event-types/{name}/schemas/{version}':
+    get:
+      tags:
+        - schema-registry-api
+      description: |
+        Retrieves a given schema version. A special `{version}` key named 'latest' is provided for
+        convenience.
+    responses:
+        '200':
+          schema:
+            type: object
+          description: |
+            Schema as JSON.
```

Comment:
@v-stepanov @rcillo is it consistent with the type we use to accept schema definition when creating an `enent-type`?

Do not we use `string` there? 

I think we should at least document the difference if there is any.